tools/etl/tg-jdbc-driver/README.md

```diff
@@ -8,18 +8,52 @@ The TigerGraph JDBC Driver is a Type 4 driver, converting JDBC calls directly in
 | --- | --- | --- | --- | --- | --- |
 | 1.0 | 2.2.4+ | 1.8 | Rest++ | JSON | Support builtin, compiled queries and loading jobs |
 | 1.1 | 2.4.1+ | 1.8 | Rest++ | ResultSet | Support tabular format and Spark |
-| 1.2 | 2.4.1+ | 1.8 | Rest++ | ResultSet | Support interpreted queries and Spark partitioning |
+| 1.2.x | 2.4.1~3.4 | 1.8 | Rest++ | ResultSet | Support interpreted queries and Spark partitioning |
 
 ## Dependency list
 | groupId | artifactId | version |
 | --- | --- | --- |
-| org.apache.commons | commons-io | 1.3.2 |
-| org.apache.httpcomponents | httpclient | 4.5.8 |
+| org.apache.commons | commons-io | 2.7 |
+| org.apache.httpcomponents | httpclient | 4.5.13 |
 | org.json | json | 20180813 |
 | org.glassfish | javax.json | 1.1.4 |
-| junit | junit | 4.11 |
+| org.junit.jupiter         | junit-jupiter-engine | 5.8.2 |
+| org.junit.vintage | junit-vintage-engine | 5.8.2 |
+| org.apache.logging.log4j | log4j-core | 2.17.2 |
+| org.apache.spark | spark-core_2.13 | 3.2.0 |
+
+## Download from Maven Central Repository
+
+Tigergraph JDBC Driver can be searched in https://search.maven.org as `tigergraph-jdbc-driver`.
+
+Please refer to [Versions compatibility](#Versions compatibility) to find the **suitable version**.
```

Comment:
The link to `Versions compatibility` is not working. You can see from here: https://github.com/pxieTG/ecosys/tree/master/tools/etl/tg-jdbc-driver